instrumentation/opentelemetry_oban/test/opentelemetry_oban_test.exs

```diff
@@ -0,0 +1,168 @@
+defmodule OpentelemetryObanTest do
```

Comment:
There was a comment by @ricardoccpaiva before moving this to this repo:
> What do you think of using https://hexdocs.pm/oban/Oban.Testing.html for testing? I believe it's a better approach because it avoids using :telemetry.execute explicitly thus being more future proof in case of changes.

We thought that it's possible to use sqlite3 but it's not. Oban is PG only. I could try setting up PG but that's a bit more effort (especially in regards to CI).